Jenkinsfile

```diff
@@ -141,6 +141,9 @@ pipeline {
     triggers {
         /* groovylint-disable-next-line AddEmptyString */
         cron(env.BRANCH_NAME == 'master' ? 'TZ=UTC\n0 0 * * *\n' : '' +
+             /* groovylint-disable-next-line AddEmptyString */
+             BRANCH_NAME.startsWith('feature/') ? '0 12 * * 0\n' : '' +
```

Comment:
I have often wondered how Jenkins reconciles these `cron()` entries when they are differing/conflicting across branches.  Indeed I am not entirely convinced that these `cron()` entries are isolated to just the branch they are added to but rather get added to some global list of entries.

Given that, I would be concerned that this change will actually mean Jenkins will run this timed testing on all current `feature/*` branches.

Further, we need to be sure this does not accidentally get landed to master when this feature branch lands.  Any thoughts on how to do that?  Should we but a big DO NOT LAND comment here or something?